src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.awt.Shape;
+import java.util.Random;
+
+public final class FigureSupplier {
+    public static final int RADIUS = 10;
+    public static final int FIGURE_COUNT = 5;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        String color = colorSupplier.getRandomColor();
+        int randomValue = random.nextInt(FIGURE_COUNT);
+        int index = random.nextInt(FIGURE_COUNT);
+        Shape shape;
```

Comment:
What are the reasons for using `awt.Shape `class here? I believe GUI development is not the purpose of the task